backend/mora/service/org.py

```diff
@@ -14,20 +14,110 @@
 
 '''
 
-import operator
-
 import flask
 import werkzeug
 
 from .. import common
 from .. import mapping
 from .. import util
+from .. import exceptions
 
 blueprint = flask.Blueprint('organisation', __name__, static_url_path='',
                             url_prefix='/service')
 
 
-def get_one_organisation(c, orgid, org=None):
+class ConfiguredOrganisation:
```

Comment:
Why is this a class?

Reply:
Mainly to hold **valid** as a class variable, so You don't need a 'global valid' inside a validator function